chain/client/tests/catching_up.rs

```diff
@@ -896,13 +891,14 @@ mod tests {
                         partial_encoded_chunk,
                     } = msg
                     {
-                        println!("S {:?} PER {:?}", sender_account_id, partial_encoded_chunk);
                         if responded.contains(&(
                             route_back.clone(),
                             partial_encoded_chunk.parts.iter().map(|x| x.part_ord).collect(),
                             partial_encoded_chunk.chunk_hash.clone(),
                         )) {
-                            println!("== SAME RESPONSE AGAIN!");
+                            if verbose {
```

Comment:
switch the `if verbose` and the outer condition here and in other places

Reply:
++